src/app.ts

```diff
@@ -40,3 +45,76 @@ export default async function newApp(): Promise<Express> {
   }
   return app;
 }
+
+function registerRoutes(app: Express): void {
+  app.get(
+    "/public-administrations",
```

Comment:
isn't there any authentication in this endpoint?

Reply:
No, at this stage no authentication has been introduced yet.